server/pom.xml

```diff
@@ -201,6 +201,21 @@
       <artifactId>httpcore</artifactId>
       <version>4.4.13</version>
     </dependency>
+    <dependency>
+      <groupId>org.jooq</groupId>
+      <artifactId>jooq</artifactId>
+      <version>3.13.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jooq</groupId>
+      <artifactId>jooq</artifactId>
+      <version>3.13.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jooq</groupId>
+      <artifactId>jooq</artifactId>
+      <version>3.13.5</version>
+    </dependency>
```

Comment:
There should be at most one dependency for a given `groupId` and `artifactId` in the same file, and rather than specifying a version it should inherit the version specified in the top-level POM--for instance, dropwizard in this file has
```
    <dependency>
      <groupId>io.dropwizard</groupId>
      <artifactId>dropwizard-auth</artifactId>
    </dependency>
```
so that it uses the dropwizard version from the top-level file. (The httpcore import should probably also be updated to do the same thing.)

I'm surprised that this dependency would be newly required, since the server code already used Jooq. Is it possible that this isn't necessary, and there was an issue with the devenv?